src/InnerImageZoom/InnerImageZoom.vue

```diff
@@ -67,7 +67,6 @@
 
```

Comment:
Harrison, would you mind looking this over and making sure I pushed this to the correct place